utils/wasm-builder/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
```

Comment:
Can you please fork the `wasm-builder` crate instead of copy-pasting its files? That way it would be easier to track the differences and submit PRs upstream if needed.

Reply:
@vkomenda I've added [a separate task MESH-1107](https://polymath.atlassian.net/browse/MESH-1107?atlOrigin=eyJpIjoiMTA0NzY2NzQyMGFmNDMwOWJiMDQzNWI2M2RlMjI1MDMiLCJwIjoiaiJ9) for that because is more time than expected.